changelog/unreleased/3440

```diff
@@ -0,0 +1,7 @@
+Enhancement: Lock delay enforced
+
+A new branding/MDM option has been added for the
```

Comment:
I have done the work of my future self, but if you think it is not necessary, I omit it.

Reply:
Yes please, we don't know who will develop this and if they will be aware that it is already on changelog, so to avoid repetitions, let's only add what we have done in this PR 👍